### DISCUSSION TITLE
Issue in Model Viewer

### DISCUSSION THREAD

**Discussion by samuelss1102** [community/community#20878] [Projects and Issues] [answered]
I can't implement the Google Model Viewer in Next.js.
Is there anyone who knows the reason?

**Comment by IceDev528** [comment, marked as answer]
const ModelViewer = dynamic(
  () => import('components/general/ModelViewer/ModelViewer'),
  { ssr: false }
);

**Comment by savvydev007** [comment]
I have met the same problem before.
As @IceDev528  Said, the point in the SSR.
Am also have done with @IceDev528 's code
